eventlet/backdoor.py

```diff
@@ -2,8 +2,6 @@
 import errno
 import socket
 import sys
-import errno
```

Comment:
errno is used many times in this module, we can't remove it. For an example, please, see line 37.

Reply:
It's still imported, see line 2 right above it. The issue was it was being imported twice.

Reply:
Oh sorry, I missed the previous import.